### PR TITLE
Refactor Kubernetes client creation

### PR DESCRIPTION
**Pull request by VineethReddy02**
use kubeClient context for connecting to kube api-server rather than creating new clients every time, Also, rename the filenames.

### CODE REVIEW

File: cli/cmd/helm/helmInstall.go [hunk 1]
```diff
@@ -14,8 +14,6 @@ import (
 	"github.com/timescale/tobs/cli/pkg/utils"
```
Comment (cevian) [inline cli/cmd/helm/helmInstall.go:14]: change file name?

File: cli/cmd/helm/helmUninstall.go [hunk 2]
```diff
@@ -55,9 +55,10 @@ func helmUninstall(cmd *cobra.Command, args []string) error {
```
Comment (cevian) [inline cli/cmd/helm/helmUninstall.go:55]: change file name?

File: cli/pkg/k8s/interface.go [hunk 3]
```diff
@@ -0,0 +1,63 @@
+package k8s
```
Comment (cevian) [inline cli/pkg/k8s/interface.go:1]: rename to client.go?

File: cli/pkg/k8s/k8s.go [hunk 4]
```diff
@@ -10,7 +10,6 @@ import (
 	"os/exec"
```
Comment (cevian) [inline cli/pkg/k8s/k8s.go:10]: rename to client_impl.go?

**Review by cevian — APPROVED** [review]
some nits but also should the filename convention be change_password.go not changePassword.go don't feel super strongly tho.

File: cli/cmd/helm/helmInstall.go (hunk 1 above)
Comment (VineethReddy02) [reply cli/cmd/helm/helmInstall.go:14]: In the upcoming PR, I would like to drop `tobs helm` (these are helm subcommands) cmd, so planning to rid of these files in that PR. :)

File: cli/cmd/grafana/portforward.go [hunk 5]
```diff
@@ -31,12 +31,13 @@ func grafanaPortForward(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("could not port-forward Grafana: %w", err)
```
Comment (cevian) [inline cli/cmd/grafana/portforward.go:31]: file port_forward.go?

File: cli/cmd/helm/helm_uninstall.go [hunk 6]
```diff
@@ -55,9 +55,10 @@ func helmUninstall(cmd *cobra.Command, args []string) error {
```
Comment (cevian) [inline cli/cmd/helm/helm_uninstall.go:55]: filename uninstall.go (drop helm_)

File: cli/cmd/port-forward/portforward.go [hunk 7]
```diff
@@ -62,42 +62,43 @@ func portForward(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("could not port-forward: %w", err)
```
Comment (cevian) [inline cli/cmd/port-forward/portforward.go:62]: filename port_forward.go

File: cli/cmd/promlens/portforward.go [hunk 8]
```diff
@@ -23,12 +23,13 @@ func init() {
 }
```
Comment (cevian) [inline cli/cmd/promlens/portforward.go:23]: filename port_forward.go

File: cli/cmd/promscale/portforward.go [hunk 9]
```diff
@@ -23,12 +23,13 @@ func init() {
 }
```
Comment (cevian) [inline cli/cmd/promscale/portforward.go:23]: filename port_forward.go

**Review by cevian — APPROVED** [review]

**Pull request merged**